recipes/zbar/all/conanfile.py

```diff
@@ -0,0 +1,105 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+
+class zbarConan(ConanFile):
+    name = "zbar"
+    license = "LGPL-2.1-only"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://zbar.sourceforge.net/"
+    topics = ("conan", "zbar", "bar codes")
+    description = "ZBar is an open source software suite for reading bar codes\
+                   from various sources, such as video streams, image files and raw intensity sensors"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "without_video": [True, False],
```

Comment:
I would prefer using all options as "with_xxx" and setting them to False.

Reply:
Yes, think positive :smile:

Reply:
I used the 'negative' approach because it is hows documented inside the `configure` script. I don'n mind changing it though.

Reply:
It's up to you. :smile: